Sources/NIOOpenSSL/PosixPort.swift

```diff
@@ -27,7 +27,9 @@ private let sysMlock: @convention(c) (UnsafeRawPointer?, size_t) -> CInt = mlock
 private let sysMunlock: @convention(c) (UnsafeRawPointer?, size_t) -> CInt = munlock
 
 // Sadly, stat has different signatures with glibc and macOS libc.
-#if os(Linux) || os(FreeBSD) || os(Android)
+#if os(Android)
+private let sysStat: (UnsafePointer<CChar>, UnsafeMutablePointer<stat>) -> CInt = stat(_:_:)
```

Comment:
With C Callback attribute `@convention(c)` modificator it says:

error: C function pointer signature '**(UnsafePointer<Int8>!, UnsafeMutablePointer<stat>!) -> Int32**' is not compatible with expected type '`@convention(c)` (UnsafePointer<CChar>, UnsafeMutablePointer<stat>) -> CInt' (aka **'@convention(c) (UnsafePointer<Int8>, UnsafeMutablePointer<stat>) -> Int32**')